pyccel/codegen/printing/ccode.py

```diff
@@ -104,6 +109,82 @@ def _traverse_matrix_indices(self, mat):
     def _print_Module(self, expr):
         return '\n\n'.join(self._print(i) for i in expr.body)
 
+    def _print_While(self,expr):
+        print(expr.test)
+        code = "while (%s)\n{" % self._print(expr.test)
```

Comment:
We should try to use the same indentation style for the braces throughout.
The following code:
```
def f():
    x = 3.0

x = 0.0

for i in range(2):
    x += 1.0
    
j = 3 
    
if (j == 2): 
    x += 1.0
elif j == 4:
    x += 1.0 
else:
    x += 1.0
    
while (j<5): 
    j+=1
```
translates to:
```
void f() {
    double x;
    x = 3.0;
}

int main(){
    double x;
    long j;
    long i;
    x = 0.0;
    for (i = 0; i < 2; i += 1) {
        x += 1.0;
    } 
    j = 3;
    if (j == 2){
        x += 1.0;
    } 
    else if (j == 4){
        x += 1.0;
    }
    else{
        x += 1.0;
    }
    while (j < 5)
    {
        j += 1;
    }
    return 0;
}
```
The for loop and the function definition use the convention `CONDITION SPACE BRACE`, the main program and the if loop use the convention `CONDITION BRACE` and the while loop uses the convention
```
CONDITION
BRACE
```

The most common conventions (https://en.wikipedia.org/wiki/Indentation_style#Brace_placement_in_compound_statements) in C/C++ are Allman (see while loop) or K&R variant 1TBS (see for and function def). Personally I prefer Allman style as it makes it easy to spot matching braces but K&R works too. The most important thing is that the same style is used throughout.

Do you have an opinion about the style @yguclu , @saidctb ?

Reply:
i don't have any preference regarding the style but we should choose one  , we probably need to make it automatic and easy to change.

Reply:
at this moment im gonna use the allman style for all the statements and the function definition

Reply:
I also don't have any preference about style, as long as it is consistently used by our C printer. As for being able to switch between styles automatically as suggested by @saidctb, that may be a good idea but I don't know how much that's going to complicate our code.

Reply:
I think we can leave switching between styles for another pull request if we want to add that.
If @nhamidn has already used Allman style everywhere then let's stick with that for now.

@nhamidn would you mind also formatting the main in the same way (see `_print_Program`)

Reply:
@EmilyBourne im gonna take care of it